### PR TITLE
Add a view for jobs creation

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by Wawrzyn321 — DISMISSED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -962,6 +941,34 @@ jobs:
...
+      parallelism: Parallelism
+      starting-deadline: Starting Deadline
+      successful-jobs-history-limit: Successful Jobs History Limit
+      suspend: Suspend
+    placeholders:
+      args: Arguments to the entrypoint.
+      command: Command to run in a container.
+      docker-image: Enter the Docker image tag, for example, 'busybox'.
+      failed-jobs-history-limit: 'The number of failed finished jobs to retain (default: 1).'
+      parallelism: 'The maximum number of pods the job should run at any given time (default: 1).'
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:963]:
```suggestion
      parallelism: 'The maximum number of Pods the job should run at any given time (default: 1).'
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -962,6 +941,34 @@ jobs:
...
+      starting-deadline: Optional deadline in seconds.
+      successful-jobs-history-limit: 'The number of successful finished jobs to retain (default: 3).'
+    tooltips:
+      args: Docker image CMD is used if arguments are not provided. Provide either args or command.
+      command: The docker image's ENTRYPOINT is used if this is not provided. Provide either args or command.
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:968]:
```suggestion
      command: The Docker image's ENTRYPOINT is used if this is not provided. Provide either args or command.
```
Should the ENTRYPOINT be capitalized? It's not written that way everywhere in the doc.

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -962,6 +941,34 @@ jobs:
...
+      command: Command
+      docker-image: Docker Image
+      failed-jobs-history-limit: Failed Jobs History Limit
+      image-pull-policy: Image Pull Policy
+      parallelism: Parallelism
+      starting-deadline: Starting Deadline
+      successful-jobs-history-limit: Successful Jobs History Limit
+      suspend: Suspend
+    placeholders:
+      args: Arguments to the entrypoint.
```
Comment (grego952) [inline core/src/i18n/en.yaml:959]:
```suggestion
      args: Arguments to the ENTRYPOINT.
```

**Review by mmitoraj — COMMENTED** [review]
Hi, I added two comments. Thanks!

File: core/src/i18n/en.yaml (hunk 2 above)
Comment (grego952) [reply core/src/i18n/en.yaml:968]: I think it's Docker specific concept and it is capitalized in the Docker docs.

**Review by mmitoraj — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**